docs/topics/forms/formsets.txt

```diff
@@ -799,6 +883,31 @@ fields/attributes of the order and deletion fields:
     <div><label for="id_form-0-pub_date">Pub date:</label><input type="text" name="form-0-pub_date" id="id_form-0-pub_date"></div>
     <div><label for="id_form-0-my_field">My field:</label><input type="text" name="form-0-my_field" id="id_form-0-my_field"></div>
 
+Method 2: Using the Formset Class
+---------------------------------
+
+Additional fields can also be easily added while using the ``FormSet`` class.
+Similar to the above method you just have to override the default
+``add_fields`` method::
+
+.. code-block:: pycon
```

Comment:
Make sure to build the docs and check that, notice this doesn't render correctly:

<img width="448" height="164" alt="Image" src="https://github.com/user-attachments/assets/d18ad74c-c15b-45b0-9f10-f2fba749792f" />